Owncloud iOs Client/Files/Preview/DetailView/DetailViewController.m

```diff
@@ -845,11 +845,7 @@ - (void)presentWhiteView {
     
     _titleLabel.text = @"";
 
-    if (k_use_open_with_UIDocumentInteractionController) {
-        [_openWith.documentInteractionController dismissMenuAnimated:YES];
-    } else {
-        [_openWith.activityPopoverController dismissPopoverAnimated:YES];
-    }
+    [_openWith.documentInteractionController dismissMenuAnimated:YES];
```

Comment:
You could check here if _openWith object exists, as you do below.

Reply:
👍